### DISCUSSION TITLE
Connecting to Debugger

### DISCUSSION THREAD

**Discussion by steventnorris** [callstack/repack#452] [Q&A]
I can't seem to find any documentation on how to connect to a debugger when using Re:Pack. When I use the typical dev menu (CMD+M on Mac) and hit debug, i get a "Failed to Connect to Debugger! Failed to connect to /10.0.2.2:8081". I do see a debugger package that seems to be included in Re:Pack, but no docs that I've found on how to use it. Anyone know?

**Comment by jbroma** [comment]
Glad you sorted out! Would you mind sharing what helped in this case? Thanks!

**Reply by steventnorris** [reply]
I just restarted the app and it started working, so I'm not entirely sure what's up with it. Assumed there was some port change or config I was missing. Process below;

- Run app. 
- Open dev menu and hit "debug" with no debugger and error occurs.
- Close app.
- Rebuild/Rerun.
- Debugger appeared and app opened in debug mode connected and working.
